### PR TITLE
fix: b/219021576

### PR DESCRIPTION
**Pull request by leahecole**
## Description

Fixes b/219021576, corresponding cl/429403697

Also adds jinja templated variables as best practice as a part of addressing #6555 

[Airflow 1 successful DAG Run](https://screenshot.googleplex.com/AwmdtnCmh94dKXE)
[Airflow 2 successful DAG Run](https://screenshot.googleplex.com/7sNuCXtQ6NvXbyp)

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Review by bradmiro — APPROVED** [review]

**Comment by leahecole** [comment]
Got my product expert, ready for @engelke now! :)

File: composer/airflow_1_samples/constraints.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 pandas-gbq==0.14.1 # must be under 0.15.0 until https://github.com/apache/airflow/issues/15113 is addressed
 SQLAlchemy==1.3.23 # must be under 1.4 until at least Airflow 2.0 (check airflow setup.py for restrictions)
-WTForms==2.3.3 # Must stay under 3.0 because of https://github.com/dpgaspar/Flask-AppBuilder/issues/1732
\ No newline at end of file
+WTForms==2.3.3 # Must stay under 3.0 because of https://github.com/dpgaspar/Flask-AppBuilder/issues/1732
+MarkupSafe==1.1.1 # as found in Airflow 1.10.15 constraints
```
Comment (dandhlee) [inline composer/airflow_1_samples/constraints.txt:4]: This is conflicting with the `markupsafe==2.0.1` pinned in the `composer/airflow_1_samples/requirements.txt` file, could you revert it on that file?

Comment (leahecole) [reply composer/airflow_1_samples/constraints.txt:4]: Ooops, great catch. I'll fix that!

Comment (leahecole) [reply composer/airflow_1_samples/constraints.txt:4]: fixed!

File: composer/airflow_1_samples/hadoop_tutorial.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
 # see https://airflow.apache.org/docs/apache-airflow/stable/timezone.html
```
Comment (dandhlee) [inline composer/airflow_1_samples/hadoop_tutorial.py:38]: Could you also update the `gce_zone` reference in this file, similar to the edits made on `composer/workflows/hadoop_tutorial.py` file?

**Review by dandhlee — APPROVED** [review]
LGTM, one minor nit below!

**Pull request merged**